proposals/low-level-struct-improvements.md

```diff
@@ -421,10 +423,12 @@ The section on `ref` field and `scoped` is long so wanted to close with a brief
 
 Detailed Notes:
 - A `ref` field can only be declared inside of a `ref struct` 
-- A `ref` field cannot be declared `static`
+- A `ref` field cannot be declared `static`, `volatile` or `const`
```

Comment:
It might be good to include some discussion on why `volatile` is not supported here.

Reply:
The reason is that `volatile` only applies when the address can be accessed from multiple threads:

> volatile. specifies that the address is a volatile address (that is, it can be referenced externally to the current thread of execution)

A `ref` field cannot be accessed in such a way though because it's a stack bound value hence it's a bit non-sensical. 

Not sure where this would be put in the doc. I'm somewhat warry of continuing to grow this section.

Reply:
> A ref field cannot be accessed in such a way though because it's a stack bound value hence it's a bit non-sensical.

I think more importantly is that it can't be accessed that way because `T*` for a struct that contains a `ref field` isn't allowed (it isn't "unmanaged"). It being a `ref struct` likewise prevents passing it around as a `void*` via `Unsafe.AsPointer`.

Some dev with enough want could use `fnptr` reinterpretation tricks to make it work and then get a `ref T` local on multiple threads, but that's probably stretching the need for language support 😄